frontend/src/app/QuickStarts.tsx

```diff
@@ -0,0 +1,56 @@
+import '@patternfly/patternfly/base/patternfly-shield-inheritable.css';
+import '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
+import '@patternfly/react-catalog-view-extension/dist/css/react-catalog-view-extension.css';
+import '@cloudmosaic/quickstarts/dist/quickstarts.css';
+import React from 'react';
+import {
+  QuickStartDrawer,
+  QuickStartContext,
+  useValuesForQuickStartContext,
+  useLocalStorage,
+} from '@cloudmosaic/quickstarts';
+
+const importAll = (r: any) => {
+  return r.keys().map(r);
+};
+
+/*
+To load a quick start from a nested component, import and use the QuickStartContext.
+Ex:
+import { QuickStartContext, QuickStartContextValues } from '@cloudmosaic/quickstarts';
+...
+const qsContext = React.useContext<QuickStartContextValues>(QuickStartContext);
+const qsButton = (
+  <button
+    onClick={() => qsContext.setActiveQuickStart && qsContext.setActiveQuickStart('template-id')}
+  >
+    Open a quick start from a nested component
+  </button>
+);
+*/
+
+export const QuickStarts = ({ children }) => {
+  // @ts-ignore
+  const allQuickStarts = importAll(require.context('../quickstarts', false, /\.ya?ml$/));
```

Comment:
This is ok for now. I'll likely move this to the backend so it can be updated without rebuilding the UI. It's likely we'll need to fetch the quickstarts from somewhere else in the future anyhow.

Reply:
Yes, another strategy (that managed kafka is using) might be to bundle documentation with the quick start drawers component into a federated module, this way if there is an update to either the documentation or the quick starts, we could deploy that separately, and on next load the host component (ODH) would fetch the updated code